src/mechanics/timeIntegration/NewmarkBase.h

```diff
@@ -29,61 +29,16 @@ class NewmarkBase : public TimeIntegrationBase
     	mMuDampingMass = rMuDampingMass;
     }
 
-    double GetDampingCoefficientMass()const
-    {
-    	return mMuDampingMass;
-    }
-
     void SetToleranceForce(double rToleranceForce)
     {
     	mToleranceForce = rToleranceForce;
     }
 
-    double GetToleranceForce()const
-    {
-    	return mToleranceForce;
-    }
-
     void SetMaxNumIterations(int rMaxNumIterations)
     {
     	mMaxNumIterations = rMaxNumIterations;
     }
 
-    int GetMaxNumIterations()const
-    {
-    	return mMaxNumIterations;
-    }
-
-    void SetNewmarkBeta(double rBeta)
-    {
-    	mBeta = rBeta;
-    }
-
-    double GetNewmarkBeta()const
-    {
-    	return mBeta;
-    }
-
-    void SetNewmarkGamma(double rGamma)
-    {
-    	mGamma = rGamma;
-    }
-
-    double GetNewmarkGamma()const
-    {
-    	return mGamma;
-    }
-
-    bool GetUseLumpedMass()const
-    {
-    	return mUseLumpedMass;
-    }
-
-    void SetUseLumpedMass(bool rUseLumpedMass)
-    {
-    	mUseLumpedMass = rUseLumpedMass;
```

Comment:
Right! But maybe, we can get rid of the option. Runge-Kutta uses it by default and Newmark is currently not using it at all, at least not in the covered tests.
https://codecov.io/gh/nutofem/nuto/src/master/src/mechanics/timeIntegration/NewmarkDirect.cpp#L161

But this is the problem with the coverage. Maybe someone uses it in his/her examples...